meilisearch/src/routes/indexes/documents.rs

```diff
@@ -100,12 +102,89 @@ pub struct GetDocument {
     retrieve_vectors: Param<bool>,
 }
 
+aggregate_methods!(
+    DocumentsGET => "Documents Fetched GET",
+    DocumentsPOST => "Documents Fetched POST",
+);
+
+#[derive(Default, Serialize)]
```

Comment:
We may prefer avoid the `Default` impl, too error prone.
```suggestion
#[derive(Serialize)]
```